### PR TITLE
feat: add admin cluster lock/unlock commands

### PR DESCRIPTION
**Pull request by benamib**
Small PR to add two new admin commands to lock & unlock a cluster

### CODE REVIEW

**Review by MacLikorne — APPROVED** [review]

**Comment by evoxmusic** [comment]
Please wait for the PR on the API side to be merged before merging this one. Thx guys.

**Comment by benamib** [comment]
This is now available in the API, can I merge this one then @evoxmusic ?

**Comment by evoxmusic** [comment]
Sure @benamib - thx

**Pull request merged**
